app/api/setup.py

```
@@ -0,0 +1,63 @@
+import os
+
+from fastapi import APIRouter, File, HTTPException, UploadFile, status
+
+from app.core import settings
+
+router = APIRouter(prefix="/api", tags=["Setup"])
+logger = logging.getLogger(__name__)
+
+
+@router.post("/setup")
+async def setup_credentials(file: UploadFile = File(...)):
```

Ruff (0.15.0)

[warning] Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)